lib/spack/spack/build_environment.py

```diff
@@ -474,6 +470,41 @@ def static_to_shared_library(static_lib, shared_lib=None, **kwargs):
     m.static_to_shared_library = static_to_shared_library
 
 
+def get_parallel_make_jobs(pkg):
+    """Returns the number of jobs to be used to build this package
+
+    Args:
+        pkg: package to be built
+    """
+    # If pkg.parallel is False, the package can't be built in parallel
+    if not pkg.parallel:
+        return 1
+
+    # Get the maximum number of build jobs, as defined in the config file
+    max_build_jobs = spack.config.get('config:max_build_jobs') or \
+        multiprocessing.cpu_count()
```

Comment:
for other stuff, we've generally hard-coded the no-config default to whatever ships in `etc/spack/defaults/config.yaml`.  Maybe continue that policy here and make it `or 16`?

Also: prefer using parens to `\`.

Reply:
Outdated